lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AuditService.java

```diff
@@ -20,30 +21,26 @@ public AuditService(AmazonSQS sqs, ConfigurationService configurationService) {
         this.queueUrl = configurationService.getSqsAuditEventQueueUrl();
     }
 
-    public void sendAuditEvent(AuditEventTypes eventType, UUID sessionId, String clientId)
-            throws SqsException {
+    public void sendAuditEvent(AuditEventTypes eventType) throws SqsException {
         try {
             SendMessageRequest sendMessageRequest =
                     new SendMessageRequest()
                             .withQueueUrl(queueUrl)
-                            .withMessageBody(generateMessageBody(eventType, sessionId, clientId));
+                            .withMessageBody(generateMessageBody(eventType));
             sqs.sendMessage(sendMessageRequest);
         } catch (JsonProcessingException e) {
             throw new SqsException(e);
         }
     }
 
-    private String generateMessageBody(AuditEventTypes eventType, UUID sessionId, String clientId)
-            throws JsonProcessingException {
+    private String generateMessageBody(AuditEventTypes eventType) throws JsonProcessingException {
         AuditEvent auditEvent = new AuditEvent();
-        Instant now = Instant.now();
-        int instant = (int) now.getEpochSecond();
-        auditEvent.setTimestamp(instant);
+        auditEvent.setTimestamp(new Date());
```

Comment:
could we drop the timestamp setter and just instantiate a new Date in the AuditEvent default constructor? I can't think of reason we wouldn't want to set the timestamp to anything other than now.

Reply:
Added to constructor